File: CMOWebApi.Models/AdminModel/MasterModel/DepartmentSetupModel.cs
using System;

namespace CMOWebApi.Models.AdminModel.MasterModel
{
    public  class DepartmentSetupModel
    {
        public long Id { get; set; }
        public Nullable<int> DepartmentCode { get; set; }
        public string Address1 { get; set; }
        public string AddressHindi1 { get; set; }
        public string Address2 { get; set; }
        public string AddressHindi2 { get; set; }
        public string Address3 { get; set; }
        public string AddressHindi3 { get; set; }
        public string FooterLine1 { get; set; }
        public string FooterLineHindi1 { get; set; }
        public string FooterLine2 { get; set; }
        public string FooterLineHindi2 { get; set; }
        public string Logo1 { get; set; }
        public string Logo2 { get; set; }
        public Nullable<bool> IsActive { get; set; } = true;
        public Nullable<bool> IsDeleted { get; set; } = false;
        public Nullable<int> OfficeCode { get; set; }
        public Nullable<long> FileSize { get; set; }
        public string FacebookUrl { get; set; }
        public string Twitter { get; set; }
        public string Youtube { get; set; }
        public string Backgroundcolor { get; set; }
        public Nullable<bool> IsAutoEmail { get; set; }
        public Nullable<bool> IsAutoSMS { get; set; }
        public string SerialNumber { get; set; }
    }

    public class DepartmentSetupViewModel
    {
        public long Id { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string FooterLine1 { get; set; }
        public string FooterLine2 { get; set; }
        public string DepartmentTitle { get; set; }
        public string OfficeName { get; set; }
		public Nullable<bool> IsActive { get; set; }
		public Nullable<bool> IsDeleted { get; set; }
	}

    public class DepartmentSetupCancellationModel
    {
        public Nullable<bool> IsAutoEmail { get; set; }
        public Nullable<bool> IsAutoSMS { get; set; }
    }

}

File: ANGULAR/ANGULAR/src/app/Shared/Model/Master/department-setup.model.ts
export class DepartmentSetupModel {
    Id: number;
    DepartmentCode: string|number;
    Address1: string;
    AddressHindi1: string;
    Address2: string;
    AddressHindi2: string;
    Address3: string;
    AddressHindi3: string;
    FooterLine1: string;
    FooterLineHindi1: string;
    FooterLine2: string;
    FooterLineHindi2: string;
    Logo1: string;
    Logo2: string;
    OfficeCode: number|string;
    FileSize: number;
    FacebookUrl: string;
    Twitter: string;
    Youtube: string;
    Backgroundcolor: string;
    SerialNumber: string;
    IsAutoEmail: boolean;
    IsAutoSMS: boolean;
}


export class DepartmentSetupViewModel {
    Id: number;
    Address1: string;
    Address2: string;
    FooterLine1: string;
    FooterLine2: string;
    DepartmentTitle: string;
    OfficeName: string;
    IsActive: boolean;
    IsDeleted: boolean;
}
